logstash-core/lib/logstash/pipeline.rb

```diff
@@ -32,24 +32,34 @@ module LogStash; class Pipeline
     :started_at,
     :thread,
     :config_str,
+    :config_hash,
     :settings,
     :metric,
     :filter_queue_client,
-    :input_queue_client
+    :input_queue_client,
+    :xopts
```

Comment:
the `xopts` name is confusing to me. What does it mean? Can we use a word or two to describe `word1_word2` or something instead of this `xopts` name?